static/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Toolchain for use with the HH Theme",
   "main": "Gruntfile.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "jambo build && grunt webpack"
```

Comment:
These don't need npx before them, right? I should use the jambo and grunt versions specified in the package.json

Reply:
no they don't need npx, I can add if you want.

Reply:
nah, its good as is